ERCS/erc-798.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: EIP-798
```

Comment:
```suggestion
eip: 7652
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

Reply:
Thanks a lot,it has been modified as suggested.